src/MCPClient/tests/test_antivirus.py

```diff
@@ -0,0 +1,211 @@
+# -*- coding: utf8 -*-
+"""Tests for the archivematicaClamscan.py client script."""
+
+import os
+import sys
+
+from collections import OrderedDict, namedtuple
+
+import pytest
+import test_clamdscan
+
+THIS_DIR = os.path.dirname(os.path.abspath(__file__))
+sys.path.append(
+    os.path.abspath(os.path.join(THIS_DIR, '../lib/clientScripts')))
+
+import archivematicaClamscan
+
+# Antivirus Tests
+
+
+def test_get_scanner(settings):
+    """ Test that get_scanner returns the correct instance of antivirus
+    per the user's configuration. Test return of clamdscanner by default. """
+
+    # doubly ensure that clamd env settings are available to the test
+    test_clamdscan.setup_clamdscanner(settings)
+
+    # clamscanner returned if specified
+    settings.CLAMAV_CLIENT_BACKEND = "clamscanner"
+    scanner = archivematicaClamscan.get_scanner()
+    assert isinstance(scanner, archivematicaClamscan.ClamScanner)
+
+    # clamdscanner returned if specified
+    settings.CLAMAV_CLIENT_BACKEND = "clamdscanner"
+    scanner = archivematicaClamscan.get_scanner()
+    assert isinstance(scanner, archivematicaClamscan.ClamdScanner)
+
+    # clamdreturned by default if something else is specified
+    settings.CLAMAV_CLIENT_BACKEND = "fprot"
+    scanner = archivematicaClamscan.get_scanner()
+    assert isinstance(scanner, archivematicaClamscan.ClamdScanner)
+
+    # empty string
+    settings.CLAMAV_CLIENT_BACKEND = ""
+    scanner = archivematicaClamscan.get_scanner()
+    assert isinstance(scanner, archivematicaClamscan.ClamdScanner)
+
+    # None
+    settings.CLAMAV_CLIENT_BACKEND = None
+    scanner = archivematicaClamscan.get_scanner()
+    assert isinstance(scanner, archivematicaClamscan.ClamdScanner)
+
+    # numerical value
+    settings.CLAMAV_CLIENT_BACKEND = 10
+    scanner = archivematicaClamscan.get_scanner()
+    assert isinstance(scanner, archivematicaClamscan.ClamdScanner)
+
+
+args = OrderedDict()
+args['file_uuid'] = 'ec26199f-72a4-4fd8-a94a-29144b02ddd8'
+args['path'] = '/path'
+args['date'] = '2019-12-01'
+args['task_uuid'] = 'c380e94e-7a7b-4ab8-aa72-ec0644cc3f5d'
+
+
+class FileMock():
+
```

Comment:
You added this blank line but flake8 seemed to be happy with it.

Reply:
Flake8 3.2.1. output with either configuration of line:

```
ross-spencer@artefactual:~/git/artefactual-labs/am/src/archivematica/src/MCPClient/tests:$ flake8 *.py
test_antivirus.py:66:1: E302 expected 2 blank lines, found 1
ross-spencer@artefactual:~/git/artefactual-labs/am/src/archivematica/src/MCPClient/tests:$ flake8 *.py
test_antivirus.py:72:1: E302 expected 2 blank lines, found 1
```

Reply:
So before and after the class right?